src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5246,7 +5248,12 @@ public void StartEvent(string eventName, EventAttribute eventAttribute)
 #endif
         }
 
-        public void AddEventParameter(Type type, string name)
+        public void AddEventParameter(
+#if !ES_BUILD_STANDALONE
+            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
I'm not sure I understand why this is necessary. This method doesn't appear to use reflection on `type` and I don't see any warnings in the suppressions file.

Reply:
Ah, I see it calls `GetTypeName` which needs `DynamicallyAccessedMemberTypes.Fields`. We should probably be matching the MemberTypes, if possible, so more things aren't preserved than are necessary.

Reply:
I don’t think this change is needed.